VisitTrackerMiddleware processing request
Recording visit from IP: 193.34.212.89
Invalid HTTP_HOST header: 'o69iay0p.hulianwgj.com:8000'. You may need to add 'o69iay0p.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'o69iay0p.hulianwgj.com:8000'. You may need to add 'o69iay0p.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'o69iay0p.hulianwgj.com:8000'. You may need to add 'o69iay0p.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'o69iay0p.hulianwgj.com:8000'. You may need to add 'o69iay0p.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
VisitTrackerMiddleware processing request
Recording visit from IP: 35.241.157.174
Invalid HTTP_HOST header: 'exilpmls.hulianwgj.com'. You may need to add 'exilpmls.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'exilpmls.hulianwgj.com'. You may need to add 'exilpmls.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
VisitTrackerMiddleware processing request
Recording visit from IP: 89.42.231.140
VisitTrackerMiddleware processing request
Recording visit from IP: 193.34.212.89
Invalid HTTP_HOST header: 'login.hulianwgj.com'. You may need to add 'login.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'login.hulianwgj.com'. You may need to add 'login.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'login.hulianwgj.com'. You may need to add 'login.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'login.hulianwgj.com'. You may need to add 'login.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
VisitTrackerMiddleware processing request
Recording visit from IP: 89.42.231.140
Invalid HTTP_HOST header: 'pingjs.qq.com'. You may need to add 'pingjs.qq.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'pingjs.qq.com'. You may need to add 'pingjs.qq.com' to ALLOWED_HOSTS.
Bad Request: /ping.js
VisitTrackerMiddleware processing request
Recording visit from IP: 204.188.197.86
VisitTrackerMiddleware processing request
Recording visit from IP: 204.188.197.86
VisitTrackerMiddleware processing request
Recording visit from IP: 204.188.197.86
Internal Server Error: /login/
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 204, in _get_response
    self.check_response(response, callback)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 332, in check_response
    raise ValueError(
ValueError: The view letters.views.login_view didn't return an HttpResponse object. It returned None instead.
VisitTrackerMiddleware processing request
Recording visit from IP: 185.218.84.178
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'bqw.hulianwgj.com'. You may need to add 'bqw.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'bqw.hulianwgj.com'. You may need to add 'bqw.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
VisitTrackerMiddleware processing request
Recording visit from IP: 89.42.231.140
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
VisitTrackerMiddleware processing request
Recording visit from IP: 220.88.38.133
VisitTrackerMiddleware processing request
Recording visit from IP: 220.88.38.133
Invalid HTTP_HOST header: 'xowzpmls.hulianwgj.com'. You may need to add 'xowzpmls.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'xowzpmls.hulianwgj.com'. You may need to add 'xowzpmls.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'tzs.hulianwgj.com'. You may need to add 'tzs.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'tzs.hulianwgj.com'. You may need to add 'tzs.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
VisitTrackerMiddleware processing request
Recording visit from IP: 89.35.130.134
Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.1.hulianwgj.com'. You may need to add 'www.1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '1.hulianwgj.com'. You may need to add '1.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
VisitTrackerMiddleware processing request
Recording visit from IP: 196.251.70.87
Not Found: /.git/refs/tags/
VisitTrackerMiddleware processing request
Recording visit from IP: 165.22.47.52
Not Found: /aaa9
VisitTrackerMiddleware processing request
Recording visit from IP: 165.22.47.52
Not Found: /aab8
VisitTrackerMiddleware processing request
Recording visit from IP: 165.22.47.52
VisitTrackerMiddleware processing request
Recording visit from IP: 104.234.115.210
VisitTrackerMiddleware processing request
Recording visit from IP: 104.234.115.210
VisitTrackerMiddleware processing request
Recording visit from IP: 104.234.115.210
Not Found: /favicon.ico
Invalid HTTP_HOST header: 'bsn.hulianwgj.com'. You may need to add 'bsn.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'bsn.hulianwgj.com'. You may need to add 'bsn.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
VisitTrackerMiddleware processing request
Recording visit from IP: 193.34.212.89
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
VisitTrackerMiddleware processing request
Recording visit from IP: 147.185.132.76
VisitTrackerMiddleware processing request
Recording visit from IP: 196.251.86.175
Not Found: /.env
VisitTrackerMiddleware processing request
Recording visit from IP: 54.164.157.105
Not Found: /favicon.ico
VisitTrackerMiddleware processing request
Recording visit from IP: 54.164.157.105
Not Found: /favicon.png
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/usr/local/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/usr/local/lib/python3.8/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.hulianwgj.com'. You may need to add 'www.hulianwgj.com' to ALLOWED_HOSTS.
Bad Request: /
VisitTrackerMiddleware processing request
Recording visit from IP: 72.50.214.106
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
Not Found: /favicon.ico
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/apps.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0008_schedule.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0007_letter_image.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0006_remove_letter_image.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0002_letter_sent_at.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0009_todo.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0010_remove_todo_user.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0011_letter_is_opened_letter_opened_at.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0012_letter_last_opened_ip_letter_opened_ips.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0013_userprofile.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0014_coupleprofile.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0015_userprofile_view_password_delete_coupleprofile.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0016_userprofile_is_online_userprofile_last_activity.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0017_userprofile_login_ip.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0005_letter_image.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0007_letter_image.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0004_letter_comment.py changed, reloading.
Watching for file changes with StatReloader
/home/admin/django/letter_recorder/songline/migrations/0003_letter_sent.py changed, reloading.
Watching for file changes with StatReloader
VisitTrackerMiddleware initialized
Watching for file changes with StatReloader
VisitTrackerMiddleware initialized
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
Forbidden: /switch-role/
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 193.34.212.89
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
VisitTrackerMiddleware processing request
Recording visit from IP: 116.232.121.205
